src/main/java/duke/ui/DukeUi.java

```diff
@@ -0,0 +1,90 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.Event;
+import duke.task.Deadline;
+
+import duke.tasklist.TaskList;
+
+/**
+ * Represents the User Interface of the Duke App.
+ * Lists the messages that are printed out when a corresponding command is executed.
+ */
+public class DukeUi {
+    public DukeUi() {
+
+    }
+
+    public void printWelcomeMessage() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+
+    public String printByeMessage() {
```

Comment:
Might want to change the `printByeMessage()` to `getByeMessage()` to be more clear on what the method is doing. Similarly for the methods below.